bt_ddos_shield/s3_manager.py

```diff
@@ -0,0 +1,9 @@
+class S3Manager:
```

Comment:
[Suggestion] make this more broad - MinerIPManager possibly so that it can be extended beyond S3